plugins/org.jkiss.dbeaver.data.office/src/org/jkiss/dbeaver/data/office/export/DataExporterXLSX.java

```diff
@@ -498,7 +498,7 @@ private CellType getCellType(DBDAttributeBinding column) {
 
     @Override
     public void exportFooter(DBRProgressMonitor monitor) throws DBException, IOException {
-        if (wb != null) {
+        if (wb != null && sheetIndex != 0) { // if any sheets are present, then sheetIndex != 0
```

Comment:
I assume the `sheetIndex` is 1-based. Maybe it would be more obvious if the condition was `sheetIndex > 0` instead?

Reply:
Ok, I can change.